tests/backup/backup_namespace_labelled_test.go

```diff
@@ -1207,3 +1207,304 @@ var _ = Describe("{ManualAndScheduleBackupUsingNSLabelWithMaxCharLimit}", func()
 		CleanupCloudSettingsAndClusters(backupLocationMap, credName, cloudCredUID, ctx)
 	})
 })
+
+// ManualAndScheduleBackupUsingNamespaceLabel Perform Namespace labeled manual and schedule backup of single and multiple namespaces along with default and custom restore
```

Comment:
How about // ManualAndScheduleBackupUsingNamespaceLabel performs?

Reply:
It is written to convey what is is doing